.env.example

```
@@ -11,9 +11,15 @@ SITE_URL=http://127.0.0.1
 # Expose the UI on this port
 UI_PORT=3000
 
+# If UI is on a different host
+#UI_HOST
+
 # Expose the API on this port
 API_PORT=8080
 
+# If API is on a different host.
+#API_HOST
```

Variables should have example values
```suggestion
#API_HOST=127.0.0.1
```

---

Can't believe I missed that, forgot to update it.